documentation/sphinx/source/tss.rst

```diff
@@ -0,0 +1,115 @@
+.. _testing-storage-server:
```

Comment:
Is there a link somewhere to this document? Also generally your testing should including building the documentation with cmake and then looking at it with your browser. You can do this by running `ninja docpreview` which starts an http server on your local machine

Reply:
I linked it from the fdb CLI page when it references configuring TSS.
Is there a higher level directory/index type thing I should link from as well?
And thanks! That makes sense